opennms-doc/releasenotes/src/asciidoc/releasenotes/whatsnew.adoc

```diff
@@ -8,8 +8,8 @@
 
 === System Requirements
 
-* *Java 8*: OpenNMS Horizon 24 requires Java 8 as the runtime environment.
-  To run Horizon 24, we recommend the most recent version of Oracle JDK 8 for your platform.
+* *Java 8 through 11*: OpenNMS Horizon 24 requires at least Java 8 as the runtime environment and now supports running on JDK 11 as well.
+  To run Horizon 24, we recommend the most recent version of Oracle JDK 8 for your platform, or the latest OpenJDK 11.
```

Comment:
Should we put a hint here, you won't get any updates anymore with Oracle JDK 8 if you don't pay for a license?